ansible/my-ansible-playbook.yml

```
@@ -0,0 +1,91 @@
+- name: Simple Playbook
+  hosts: managed_nodes
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: secret_key
+      prompt: Input the SECRET KEY
+
+    - name: base_api_url
+      prompt: Input the TRELLO BASE API
```

Nothing wrong with this, but given that this is likely to be the same for every environment, and isn't secret, would it make sense to set a default for this?
```suggestion
    - name: base_api_url
      prompt: Input the TRELLO BASE API
      default: https://api.trello.com/1/
```